buildSrc/src/main/groovy/org/opensearch/hadoop/gradle/fixture/OpenSearchFixturePlugin.groovy

```diff
@@ -19,10 +19,10 @@
 
 package org.opensearch.hadoop.gradle.fixture
 
-import org.elasticsearch.gradle.testclusters.ElasticsearchCluster
-import org.elasticsearch.gradle.testclusters.StandaloneRestIntegTestTask
-import org.elasticsearch.gradle.testclusters.TestClustersPlugin
-import org.elasticsearch.gradle.testclusters.TestDistribution
+import org.opensearch.gradle.testclusters.OpenSearchCluster
```

Comment:
Looks like none of the testclusters jars are in the snapshot library. What is the course of action here @nknize?